dist/systemd/system/zincati.service

```diff
@@ -1,6 +1,9 @@
 [Unit]
 Description=Zincati Update Agent
 Documentation=https://github.com/coreos/zincati
+After=network.target
+Requires=boot-complete.target
```

Comment:
I feel like the `boot-complete.target` ones probably warrant a comment like:

```
# Make sure to start *after* a successful boot, so broken updates don't keep rolling out in cluster contexts
```

?

Reply:
Indeed. Added a few explanatory words.